src/utils/breach-resolution.js

```diff
@@ -100,18 +106,25 @@ const breachResolutionDataTypes = {
  * The checklist serves the UI with relevant recommendations based on the array of datatypes leaked during a breach.
  *
  * @param {Array} userBreachData contains monitored verified emails array. Each email may contain a breaches array
+ * @param {Partial<{ countryCode: string }>} options
  * @returns {*} void
  */
-function appendBreachResolutionChecklist (userBreachData) {
+function appendBreachResolutionChecklist (userBreachData, options = {}) {
   const { verifiedEmails } = userBreachData
   for (const { breaches } of verifiedEmails) {
     breaches.forEach(b => {
       const dataClasses = b.DataClasses
       const args = {
         companyName: b.Name,
-        breachedCompanyUrl: `https://${b.Domain}`
+        breachedCompanyLink: `<a href="https://${b.Domain}" target="_blank">${b.Domain}</a>`,
+        firefoxRelayLink: `<a href="https://relay.firefox.com" target="_blank">${getMessage('breach-checklist-link-firefox-relay')}</a>`,
+        passwordManagerLink: `<a href="https://www.mozilla.org/firefox/features/password-manager/" target="_blank">${getMessage('breach-checklist-link-password-manager')}</a>`,
+        mozillaVpnLink: `<a href="https://www.mozilla.org/products/vpn/" target="_blank">${getMessage('breach-checklist-link-mozilla-vpn')}</a>`,
+        equifaxLink: `<a href="https://www.equifax.com/personal/credit-report-services/credit-freeze/" target="_blank">${getMessage('breach-checklist-link-equifax')}</a>`,
+        experianLink: `<a href="https://www.experian.com/freeze/center.html" target="_blank">${getMessage('breach-checklist-link-experian')}</a>`,
+        transUnionLink: `<a href="https://www.transunion.com/credit-freeze" target="_blank">${getMessage('breach-checklist-link-transunion')}</a>`
```

Comment:
Possibly more of a PM question, but should we be concerned about maintaining these specific external links?  Do we have a good means of detecting whether any of them change?

Reply:
Called that out on the doc; I'm indeed not sure how faithful these companies are [to TimBL](https://www.w3.org/Provider/Style/URI.html) :P